clib/hMap.ml

```diff
@@ -38,6 +38,8 @@ struct
   let is_empty = Int.Map.is_empty
 
   let mem x s =
+    if Int.Map.is_empty s then false
```

Comment:
So this tiny change is enough to remove the slowdown in hott/category theory
but presumably if they add anything to the transparent state that won't work anymore